apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -474,4 +474,13 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 			margin-bottom: 0;
 		}
 	}
+
+	// Adjust template title.
+	.editor-post-title__input {
```

Comment:
Is this class added so it intentionally inherits existing styles? If not and these styles are bespoke to this one block then is the classname specific enough?